src/status.js

```diff
@@ -0,0 +1,25 @@
+export function status(arr) {
+  arr.forEach((element) => {
+    const checkbox = document.getElementById(`${element.index}-checkbox`);
+    checkbox.addEventListener('change', () => {
+      if (checkbox.checked) {
+        element.completed = true;
+        window.localStorage.setItem('tasklist', JSON.stringify(arr));
+      } else {
+        element.completed = false;
+        window.localStorage.setItem('tasklist', JSON.stringify(arr));
```

Comment:
- [ ]  standarized the localstorage function

Looks like the localStorage manipulation is very common in your code, i suggest to you create a function to set data to it without the need of writing ('tasklist') or JSON.stringify, getting the needed variables

the function will do your code more redeable and maintenaible

Reply:
Thank you for your comment. Actually, the local storage manipulation appears only in lines 7 and 10 (status.js). Do you think creating a separate function for setting items in local storage is really necessary in this case?

Reply:
Yeah, you are right i think is no needed, sorry for the interruptions